test/java/org/opendatakit/briefcase/transfer/TransferFormsTest.java

```diff
@@ -42,4 +62,27 @@ public void forms_from_varargs_factory_can_be_cleared() {
     forms.clear();
     assertThat(forms.isEmpty(), is(true));
   }
+
+  @Test
+  public void loads_all_forms() {
+    TransferForms transferForms = TransferForms.empty();
+    transferForms.load(forms);
+    assertEquals(10, transferForms.size());
```

Comment:
Why not test that the event `onChange` is triggered when forms are loaded, which is also part of this method's behavior?